examples/escrow/createIDLType.ts

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
```

Comment:
I don't think I have push access to this branch, so can you move this into `tests/escrow` and not the `examples/` dir, which is meant to be a directory with higher quality examples that follow best practices.